### PR TITLE
Update qid-batch-runner yml for new rabbitmq operator

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

For the new rabbitmq operator we need to update the yaml to use the new host

# What has changed

- Yaml file uses new rabbit host

# How to test?

Run with other branches on ticket and run the ATs pointing to this branch

# Links

[Trello](https://trello.com/c/KSy7A1CK/)

# Screenshots (if appropriate):

### CODE REVIEW

File: qid-batch-runner.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
           stdin: true
           env:
             - name: RABBITMQ_SERVICE_HOST
-              value: rabbitmq
+              value: rm-rabbitmq-client
```
Comment (NickGrantX) [inline qid-batch-runner.yml]: I think this is wrong. It's `rm-rabbitmq` everywhere else

**Review by NickGrantX — CHANGES_REQUESTED** [review]
See comment

File: qid-batch-runner.yml (hunk 1 above)
Comment (apolloakora) [reply qid-batch-runner.yml]: Nick that is correct - it has now been updated.

**Review by AdamHawtin — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
Looks good

**Pull request merged**